### ISSUE TITLE
Is this line for computing the KL loss equivalent to loss = solutions[-2]

### ISSUE THREAD

**Issue by luo-lorry** [Waffle-Liu/STRODE#1]
https://github.com/Waffle-Liu/STRODE/blob/47c96f5bd6416b2b691baf79ef4a54369b64e26b/rotatingMNIST/model.py#LL250C1-L250C38
```
first_item = solutions[-1]
# Gm_eps = torch.abs(solutions[-2]-solutions[-1])
Gm_eps = solutions[-2]-solutions[-1]
second_item = Gm_eps

loss = first_item+second_item
```

**Comment by Waffle-Liu** [comment]
您好！我已收到您的邮件，会尽快处理。谢谢！
